controls/SV-221584.rb

```diff
@@ -21,4 +21,10 @@
   tag legacy: ["SV-57639","V-44805"]
   tag cci: ["CCI-002605"]
   tag nist: ["SI-2 c"]
+
+  version = input('google_chrome_version')
+  # ???
+  describe version do
+    it { should cmp >= 74.0.0 }
```

Comment:
When compared against `74.0.0` directly, it will errors `unexpected fraction part after numeric literal`. Consider surrounding the version number with single quote like so `'74.0.0'`.

Reply:
There is a way to find google chrome version on powershell: `(Get-Item (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe').'(Default)').VersionInfo.ProductVersion`, taken from this link: https://stackoverflow.com/questions/44975653/find-version-of-chrome-browser-in-registry

Reply:
Like this?

Reply:
We would need to surround the version number with single quotes to compare correctly